app/api/connectors/BaseDownstreamConnector.scala

```diff
@@ -70,7 +68,11 @@ trait BaseDownstreamConnector extends Logging {
       http.PUT(getBackendUri(uri), body)
     }
 
-    doPut(getBackendHeaders(uri, hc, correlationId, jsonContentTypeHeader))
+    intent match {
+      case Some(x) =>doPut(getBackendHeaders(uri, hc, correlationId, jsonContentTypeHeader, ("intent", x)))
+      case _ => doPut(getBackendHeaders(uri, hc, correlationId, jsonContentTypeHeader))
```

Comment:
Might just be preference, but could we put `case None` here instead? As it should still be exhaustive case matching as it can only be a Some or None. Just reads nicer imo.

Reply:
Sure